mpu6500_spi_node:
  ros__parameters:
    device: "/dev/spidev0.0"
    frame_id: "imu_link"
    rate_hz: 200
    spi_speed_hz: 1000000
    gyro_range_dps: 2000      # 250|500|1000|2000
    accel_range_g: 16         # 2|4|8|16
    dlpf_cfg: 3               # 0..6
    smplrt_div: 4
    gyro_variance: 0.02
    accel_variance: 0.04
